src/client/app/components/MultiCompareChartComponent.jsx

```diff
@@ -6,21 +6,22 @@ import React from 'react';
 import CompareChartContainer from '../containers/CompareChartContainer';
 
 export default function MultiCompareChartComponent(props) {
-	// Compute how much space should be used in the bootstrap grid system
-	let size;
-	if (props.selectedMeters.length + props.selectedGroups.length === 1) {
-		size = 12;
-	} else if (props.selectedMeters.length + props.selectedGroups.length === 2) {
-		size = 6;
-	} else {
-		size = 4;
-	}
-
 	const centeredStyle = {
 		marginTop: '20%',
 		textAlign: 'center'
 	};
 
+	const flexContainerStyle = {
+		display: 'flex',
+		flexFlow: 'row wrap',
+		flexShrink: '3'
+	};
+
+	const flexChildStyle = {
+		width: '30%',
+		flexGrow: '1'
```

Comment:
Change this line to `flexGrow: props.selectedMeters.length + props.selectedGroups.length > 3 ? '0' : '1'`